seacrowd/sea_datasets/thai_nner/thai_nner.py

```diff
@@ -0,0 +1,202 @@
+# limitations under the License.
+
+"""
+Thai NNER Dataset
+"""
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.common_parser import load_conll_data
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+
+_CITATION = """\
+@inproceedings{buaphet-etal-2022-thai,
+    title = "{T}hai Nested Named Entity Recognition Corpus",
+    author = "Buaphet, Weerayut  and
+      Udomcharoenchaikit, Can  and
+      Limkonchotiwat, Peerat  and
+      Rutherford, Attapol  and
+      Nutanong, Sarana",
+    editor = "Muresan, Smaranda  and
+      Nakov, Preslav  and
+      Villavicencio, Aline",
+    booktitle = "Findings of the Association for Computational Linguistics: ACL 2022",
+    month = may,
+    year = "2022",
+    address = "Dublin, Ireland",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2022.findings-acl.116",
+    doi = "10.18653/v1/2022.findings-acl.116",
+    pages = "1473--1486",
+    abstract = "This paper presents the first Thai Nested Named Entity Recognition (N-NER) dataset. Thai N-NER consists of 264,798 mentions, 104 classes, and a maximum depth of 8 layers obtained from 4,894 documents in the domains of news articles and restaurant reviews. Our work, to the best of our knowledge, presents the largest non-English N-NER dataset and the first non-English one with fine-grained classes. To understand the new challenges our proposed dataset brings to the field, we conduct an experimental study on (i) cutting edge N-NER models with the state-of-the-art accuracy in English and (ii) baseline methods based on well-known language model architectures. From the experimental results, we obtained two key findings. First, all models produced poor F1 scores in the tail region of the class distribution. There is little or no performance improvement provided by these models with respect to the baseline methods with our Thai dataset. These findings suggest that further investigation is required to make a multilingual N-NER solution that works well across different languages.",
+}
+"""
+
+
+_DATASETNAME = "ThaiNNER"
+
+
+_DESCRIPTION = """\
+Thai N-NER consists of 264,798 mentions, 104 classes, and a maximum depth of 8 layers obtained from 4,894 documents in the domains of news articles and restaurant reviews. 
+To create the dataset, the authors gather 4,894 documents from two different domains: news articles and restaurant reviews. In particular, we obtain 4,396 news articles from Prachathai, a news website, and 498 restaurant reviews from Wongnai, a crowd-sourced restaurant review platform.
+
+"""
+
+
+_HOMEPAGE = "https://github.com/vistec-AI/Thai-NNER"
+
+
+_LANGUAGES = ["tha"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+
+_LICENSE = Licenses.CC_BY_SA_3_0.value
+
+
+_LOCAL = False
+
+
+_URLS = {
```

Comment:
There is a mismatch in the data split.
In any case, did you combine `train` set and `dev` (validation) set as `train` set?
The number in `train` set here is the same as `train` + `dev` in the paper.

Could you recheck the `_URL` and add `dev`